.eslintrc.json

```diff
@@ -42,11 +42,11 @@
         "props": false
       }
     ],
-    "complexity": [ 2, 3 ],
-    "max-depth": [2, 3],
-    "max-len": [1, 160, 4],
+    "complexity": [ 2, 10 ],
```

Comment:
Good question @mitogh... We have to refactor the gravity forms component. I didn't do it because I didn't want to break anything... Maybe we can add an issue to do it in the near future, what do you think?